csharp/extractor/Semmle.Extraction.CSharp/Entities/Expressions/VariableDeclaration.cs

```diff
@@ -11,10 +11,15 @@ class VariableDeclaration : Expression
     {
         VariableDeclaration(IExpressionInfo info) : base(info) { }
 
-        public static VariableDeclaration Create(Context cx, ISymbol symbol, Type type, Extraction.Entities.Location exprLocation, Extraction.Entities.Location declLocation, bool isVar, IExpressionParentEntity parent, int child)
+        public static VariableDeclaration Create(Context cx, ISymbol symbol, Type type, TypeSyntax mention, Extraction.Entities.Location exprLocation, Extraction.Entities.Location declLocation, bool isVar, IExpressionParentEntity parent, int child)
```

Comment:
Can we make it clear that `mention` is maybe null. Perhaps rename to `optionalMention` or add some doc comment.